pkg/util/testing/wrappers.go

```diff
@@ -499,3 +511,66 @@ func (rc *RuntimeClassWrapper) PodOverhead(resources corev1.ResourceList) *Runti
 func (rc *RuntimeClassWrapper) Obj() *nodev1.RuntimeClass {
 	return &rc.RuntimeClass
 }
+
+type limitRangeBuider struct{ corev1.LimitRange }
+
+func MakeLimitRange(name, namespace string) *limitRangeBuider {
+	return &limitRangeBuider{
+		LimitRange: corev1.LimitRange{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: namespace,
+			},
+			Spec: corev1.LimitRangeSpec{
+				Limits: []corev1.LimitRangeItem{
+					{
+						Type: corev1.LimitTypeContainer,
+						Min: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("2"),
+							corev1.ResourceMemory: resource.MustParse("200M"),
+						},
+						DefaultRequest: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("3"),
+							corev1.ResourceMemory: resource.MustParse("300M"),
+						},
+						Default: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("4"),
+							corev1.ResourceMemory: resource.MustParse("400M"),
+						},
+						Max: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("5"),
+							corev1.ResourceMemory: resource.MustParse("500M"),
+						},
```

Comment:
Remove these lines. It's too many constants hidden in just `MakeLimitRange`.

When we put some constants in other wrappers, they correspond to defaults or things that almost every test should need.

Reply:
done